pipelined/bevy_render2/src/render_component.rs

```diff
@@ -127,6 +150,7 @@ where
     }
 }
 
+/// Implements [`ExtractComponent`] for all asset handles.
```

Comment:
I think we should do a pass to remove "obvious" docs like this that just rephrase type information (especially for trait impls like this). I think they have net negative value because they take up space